.tugboat/.env.j2

```
@@ -24,3 +24,10 @@ GITHUB_REPO={{ TUGBOAT_GITHUB_REPO }}
 GITHUB_COMMENT_TYPE=pr
 GITHUB_PR_ISSUE_NUMBER={{ TUGBOAT_GITHUB_PR | default('') }}
 GITHUB_REF={{ TUGBOAT_PREVIEW_SHA }}
+
+# pass tags for testing PIV based on environment
+{% if TUGBOAT_GITHUB_PR == '' %}
+  CYPRESS_TAGS="not @ignore and not @piv"
+{% else %}
+  CYPRESS_TAGS="not @ignore and not @no_piv"
```

Making my brain hurt with the double negative.  :)   
I am sure there is good reason for it that I am not seeing (relatively new to cypress tests) but would a more readable 
```suggestion
  CYPRESS_TAGS="not @ignore and @piv"
```
work?

---

unfortunately no. cypress runs through all tests by default, and because these two log in tests are binary in nature (piv on or off) we have to be explicit about which one we are ignoring, rather than which one we want to run.

would changing `@no_piv` to `@piv_free` or something that isn't as double-negative-y make more sense?

---

went with `@piv_off` for now